docs/versioning_test.py

```diff
@@ -0,0 +1,90 @@
+# ---
+# jupyter:
+#   kernelspec:
+#     name: s3-specs
+#     display_name: S3 Specs
+#   language_info:
+#     name: python
+# ---
+
+# # Versionamento
+
+# O que é o Versionamento?
+
+# O versionamento é uma funcionalidade oferecida por sistemas de armazenamento de objetos, como o Magalu Cloud, 
+# que permite manter múltiplas versões de um mesmo objeto dentro de um bucket. Isso significa que, ao habilitar o versionamento,
+# você pode recuperar, restaurar ou excluir versões anteriores de arquivos, garantindo uma trilha de auditoria completa 
+# e prevenindo perdas acidentais de dados.
+
+# No Magalu Cloud, o versionamento é inspirado no funcionamento do Amazon S3, 
+# oferecendo um mecanismo robusto de controle sobre a evolução de objetos dentro de um bucket.
+
+import logging
+import pytest
+from s3_helpers import run_example
+from botocore.exceptions import ClientError
+
+# + tags=["parameters"]
```

Comment:
the params are available at params.example.yaml and they are automatically retrieved by s3_client